panel/src/components/Forms/Blocks/Block.vue

```diff
@@ -158,6 +158,9 @@ export default {
 
 			return "k-block-type-default";
 		},
+		isCollapsable() {
+			return this.fieldset.preview === "fields";
```

Comment:
`Suggestion:` Instead of this check, I would make the two methods dependent on the presence of a collapse/expand function:

```js
collapse() {
  this.$refs.editor?.collapse?.();
}
```